serverless/src/util/authorizer/validateToken.js

```diff
@@ -48,8 +48,8 @@ export const validateToken = async (jwtToken, earthdataEnvironment) => {
       } = verifiedJwtToken
 
       // Retrieve the authenticated users' access tokens from the database
-      const existingUserTokens = await dbConnection('user_tokens')
-        .select([
+      const mostRecentToken = await dbConnection('user_tokens')
+        .first([
```

Comment:
Good change we're doing the same thing but, we should offload to SQL to optimize